special-pages/pages/new-tab/app/customizer/integration-tests/customizer.page.js

```diff
@@ -367,6 +366,10 @@ export class CustomizerPage {
 
     async orderOfColorsMatchesMacos() {
         const { page } = this.ntp;
+        const subscreen = page.locator('aside').locator('[data-sub="color"]');
+
+        // wait for the elements to show, before getting any values
+        await subscreen.locator('[role=radio][data-value]').nth(0).waitFor();
```

Comment:
There was a flaky test here, I was scraping some values too early (we have deferred rendering in certain places)